<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookWidget</class>
 <widget class="QWidget" name="BookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bookTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN :</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookISBN">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Publich date :</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookPublishDate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Publich Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre :</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookGenre">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pages :</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookPages">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pages</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Authors :</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="bookAuthors">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
